modules/caddyhttp/reverseproxy/selectionpolicies.go

```diff
@@ -699,8 +705,10 @@ func (s CookieHashSelection) Select(pool UpstreamPool, req *http.Request, w http
 //
 //	lb_policy cookie [<name> [<secret>]] {
 //		fallback <policy>
+//		[max_age <duration>]
 //	}
 //
+// `max_age` is a positive (non-zero) `caddy.Duration`
```

Comment:
I don't think this needs saying tbh, it's implied by the `<duration>`. We'll document it as such in the docs website as well.

```suggestion
```